### PR TITLE
aruha-1335: take all timelines not only active

### PR DESCRIPTION
**Pull request by adyach**
[ARUHA-1335: Not possible to delete event type](https://techjira.zalando.net/browse/ARUHA-1335)

## Review
- [ ] Tests
- [ ] Documentation
- [ ] CHANGELOG

### CODE REVIEW

**Comment by v-stepanov** [comment]
@adyach 
Do you want to create a test that tests this case?
I think the reason why we have this bug is that we don't have a test for this method (or the test misses something)

File: src/acceptance-test/java/org/zalando/nakadi/webservice/EventTypeAT.java [hunk 1]
```diff
@@ -289,6 +289,19 @@ public void whenUpdateETAuthObjectThen422() throws Exception {
...
 
+    @Test
+    public void shouldDeleteEventTypeWhenOneTimelineWasDeleted() throws Exception {
+        final EventType eventType = NakadiTestUtils.createEventType();
+        NakadiTestUtils.switchTimelineDefaultStorage(eventType);
+
+        final List<Timeline> timelines = TIMELINE_REPOSITORY.listTimelinesOrdered(eventType.getName());
+        final Timeline timeline = timelines.get(0);
+        timeline.setDeleted(true);
+        TIMELINE_REPOSITORY.updateTimelime(timeline);
```
Comment (v-stepanov) [inline src/acceptance-test/java/org/zalando/nakadi/webservice/EventTypeAT.java]: but this will not update timeline in cache, right?

**Comment by v-stepanov** [comment]
👍

**Comment by adyach** [comment]
👍

**Comment by adyach** [comment]
deploy validation please

**Pull request merged**